examples/device/CMakePresets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": 6,
+  "include": [
+    "../../../hw/bsp/BoardPresets.json"
```

Comment:
path to BoardPresets.json is not correct (one `../` less)

Reply:
Nice spot, forgot this case.